src/pages/components/Demo/Demo.tsx

```diff
@@ -1,19 +1,25 @@
+/* eslint-disable max-len */
 import React, { useState, useEffect } from 'react';
-import PropTypes from 'prop-types';
 import { Button } from '@chakra-ui/react';
 import omit from 'lodash/omit';
 import { useTranslation } from 'react-i18next';
-import queryString from 'query-string';
+import queryString, { ParsedQuery } from 'query-string';
 import JSONPretty from 'react-json-pretty';
 import { Input, Checkbox } from 'antd';
 import { API_ROUTE, DICTIONARY_APP_URL } from '../../../siteConstants';
+import Word from '../../../models/interfaces/Word';
 
-const Demo = ({ searchWord, words }) => {
+interface DemoPropsInterface {
+  searchWord: string;
+  words: Word[];
+}
+
+const Demo = ({ searchWord, words }: DemoPropsInterface) => {
   const [isLoading, setIsLoading] = useState(true);
   const [isSearchingWord, setIsSearchingWord] = useState(false);
-  const [keyword, setKeyword] = useState(searchWord || '');
+  const [keyword, setKeyword] = useState<string | string[]>(searchWord || '');
```

Comment:
if `searchWord` is always expected to be a string, why are we supporting the `string[]` type? let's just have:

```ts
useState(searchWord || '')
```